lib/admin_plugins/futuretreatments.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+var futuretreatments = {
+  name: 'futuretreatments'
+  , label: 'Remove future treatments from mongo database'
+  , pluginType: 'admin'
+};
+
+function init() {
+  return futuretreatments;
+}
+
+module.exports = init;
+
+futuretreatments.actions = [
+    {
+        name: 'Find and remove treatments in the future'
+      , description: 'This task find and remove treatments in the future.'
+      , buttonLabel: 'Remove treatmens in the future'
+    }
+  ];
+
+futuretreatments.actions[0].init = function init(client, callback) {
+  var translate = client.translate;
+  var $status = $('#admin_' + futuretreatments.name + '_0_status');
+  
+  function showTreatments(treatments, table) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (8)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/983074590/issues/source?bid=2452379&fileBranchId=2865970#l27)